### PR TITLE
Fixed support for additional seaborn images

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
## Before submitting checklist

- [X] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.12" ]
+        python-version: [ "3.8", "3.12" ]
```
Comment (Raalsky) [inline .github/workflows/e2e.yml:35]: Why? We haven't changed our minimum Python version requirements for Python client.

Comment (SiddhantSadangi) [reply .github/workflows/e2e.yml:35]: The figure class has a different name in py3.7. I can update the condition to handle both, but should we add debt to support a legacy python version?

Comment (Raalsky) [reply .github/workflows/e2e.yml:35]: It doesn't sound to be a huge one so why not? 😉; at least such a small change shouldn't impact our matrix. You've could think to skip the tests for particular OS/Python combination.

Comment (SiddhantSadangi) [reply .github/workflows/e2e.yml:35]: @PatrykGala - are we good to exclude py3.7?
In that case, I'll update the changelog to mention that we are officially dropping support for py3.7

Comment (SiddhantSadangi) [reply .github/workflows/e2e.yml:35]: As discussed internally, dropping py3.7 support

**Review by normandy7 — DISMISSED** [review]

**Review by normandy7 — DISMISSED** [review]

**Review by PatrykGala — APPROVED** [review]

**Review by AleksanderWWW — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## neptune 1.12.0
+
+### Changes
+- Dropped support for Python 3.7 ([#1864](https://github.com/neptune-ai/neptune-client/pull/1864))
+
+### Fixes
+- Fixed support for additional seaborn figure types ([#1864](https://github.com/neptune-ai/neptune-client/pull/1864))
```
Comment (normandy7) [inline CHANGELOG.md:7]:
```suggestion
- Fixed support for additional Seaborn figure types ([#1864](https://github.com/neptune-ai/neptune-client/pull/1864))
```

**Review by normandy7 — APPROVED** [review]

**Pull request merged**